### DISCUSSION TITLE
Build openssl for iOS with shared flag doesn't create libssl.dylib and libcrypto.dylib

### DISCUSSION THREAD

**Discussion by danbey** [openssl/openssl#25321] [Q&A] [answered]
Hi, 
I run the following commands:
"config shared enable-fips --install=somepath"
"make"
"make install"

This will not create the  libssl.dylib and libcrypto.dylib. 

Only after changing the 15-ios.conf file and remove from the ios-common the "shared" in the disable line , i get the libssl.dylib and libcrypto.dylib. 

Is it a bug ? Or it's writen in the manual/readme that we need to remove the "shared" from the 15-ios.conf ? 

Thanks,
Dan

**Comment by t8m** [comment, marked as answer]
This was already changed on the master branch - what becomes 3.4.0.
We do not plan to backport this change to stable branches.
